pages/index.js

```diff
@@ -7,6 +7,15 @@ import ShoppingList from "../components/ShoppingList";
 export default function Home() {
   const [shoppingListItems, setShoppingListItems] = useState(SHOPPING_LIST_DB);
 
+  function toggleItemCheckedState(id) {
+    const updatedtems = shoppingListItems.filter((item) => item.id !== id);
```

Comment:
You have a little typo here. updatedItems? It's not really a problem, but maybe you want to correct it?